packages/chart-advisor/src/advisor/index.ts

```diff
@@ -15,13 +15,21 @@ import type { Advice, AdvicesWithLog, DataRows, GraphData as GData, Data } from
 
 export type AdviseParams = ChartAdviseParams | GraphAdviseParams;
 
+/**
+ * Field Type
+ * @public
+ */
+export declare type TypeSpecifics = 'null' | 'boolean' | 'integer' | 'float' | 'date' | 'string';
```

Comment:
u could import it from `@antv/data-wizard`'s analyzer

Reply:
Done